wpilib/wpilib/analogaccelerometer.py

```diff
@@ -12,6 +12,7 @@
 from .interfaces import PIDSource
 from .livewindow import LiveWindow
 from .livewindowsendable import LiveWindowSendable
+from networktables.networktable import NetworkTable
```

Comment:
Just `from networktables import NetworkTable` should be fine. But this is fine too I guess.